src/Umbraco.Cms.ManagementApi/ManagementApiComposer.cs

```diff
@@ -35,7 +35,9 @@ public void Compose(IUmbracoBuilder builder)
 
         builder
             .AddNewInstaller()
-            .AddUpgrader();
+            .AddUpgrader()
+            .AddServices()
+            .AddFactories();
```

Comment:
Single entry points for factories and services will eventually grow rather large and it'll be hard to figure out which factories and services are used for what. 

I believe `AddServices()` and `AddFactories()` should be replaced by one or more Examine dedicated `IUmbracoBuilder` extensions - i.e. `AddExamineManagement()`  (like `AddNewInstaller()` and `AddUpgrader()`).